README.md

```diff
@@ -1,6 +1,8 @@
 ![mqtt.js](https://raw.githubusercontent.com/mqttjs/MQTT.js/137ee0e3940c1f01049a30248c70f24dc6e6f829/MQTT.js.png)
 =======
 
+Allow setting "dup" flag when publishing a message
+
```

Comment:
this would need to be moved in the publish section of the README, just like all the other options.